assets/runtime/functions

```diff
@@ -310,6 +310,12 @@ owncloud_configure_domain() {
   sed -i "s|0 => '.*',|0 => '${OWNCLOUD_FQDN}',|" ${OWNCLOUD_APP_CONFIG}
 }
 
+owncloud_configure_apps_paths() {
+  echo "Configuring ownCloud::apps_paths..."
+  apps_paths="array ( 0 => array ( 'path' => OC::\$SERVERROOT.'/apps', 'url' => '/apps', 'writable' => false, ), 1 => array ( 'path' => OC::\$SERVERROOT.'/apps2', 'url' => '/apps2', 'writable' => true, ),)"
```

Comment:
Seems like at the moment we can only use this spaghetti like line, otherwise the `sed` command in `owncloud_set_param` throws an error.
(This was my attempt: https://github.com/pc-fan/docker-owncloud/blob/764c15cb26752d088e0b2337771f706457d31d07/assets/runtime/functions#L315)